prime-router/src/main/kotlin/azure/ValidateFunction.kt

```diff
@@ -178,4 +209,28 @@ class ValidateFunction(
             )
             .build()
     }
+
+    /**
+     * Return [TopicSender] for a given schema if that schema exists. This lets us wrap the data needed by
+     * processRequest without making changes to the method
+     * @param schemaName the name or path of the schema
+     * @param format the message format that the schema supports
+     * @return TopicSender if schema exists, null otherwise
+     */
+    internal fun getDummySender(schemaName: String, format: String): TopicSender? {
+        val schema = workflowEngine.metadata.findSchema(schemaName)
+        return if (schema != null) {
+            TopicSender(
+                "ValidationSender",
+                "Internal",
+                Sender.Format.valueOf(format),
+                CustomerStatus.TESTING,
+                schemaName,
```

Comment:
Is there any possible issue of validating a custom schema error messages through this?  Like if someone makes a call to `/validateWithSchema?schema=hl7%2fhcintegrations-covid-19&format=HL7` without being part of the `hcintegrations` Organization.  I'd suppose not since everything's public source and there really couldn't possibly be a nefarious use case for this, but just thought I'd ask!

Reply:
Correct, we concluded that this would be OK. We are also OK with a user calling validateWithClient and "guessing" clients. There is no personal info at risk.